charts/gardener/gardenlet/charts/runtime/templates/deployment.yaml

```diff
@@ -21,6 +21,10 @@ spec:
     metadata:
       annotations:
         checksum/configmap-gardenlet-config: {{ include (print $.Template.BasePath "/configmap-componentconfig.yaml") . | sha256sum }}
+        {{- if .Values.global.gardenlet.metrics.enabled }}
```

Comment:
Out of interest: Why do we have this condition here? In which cases would I want to disable it?

Reply:
I made it configurable in case we don't want to scrape the gardenlet for some reason. I can't think of any good reasons to do this so I removed it.